GVFS/GVFS.UnitTests/Common/GVFSEnlistmentHealthTests.cs

```diff
@@ -0,0 +1,250 @@
+﻿using GVFS.Common;
+using GVFS.Tests.Should;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace GVFS.UnitTests.Common
+{
+    [TestFixture]
+    public class GVFSEnlistmentHealthTests
+    {
+        private List<string> fauxGitFiles = new List<string>();
+        private List<string> fauxGitFolders = new List<string>();
+        private List<string> fauxPlaceholderFiles = new List<string>();
+        private List<string> fauxPlaceholderFolders = new List<string>();
+        private List<string> fauxModifiedPathsFiles = new List<string>();
+        private List<string> fauxModifiedPathsFolders = new List<string>();
+        private List<string> fauxSkipWorktreeFiles = new List<string>();
+        private GVFSEnlistmentHealthCalculator enlistmentHealthCalculator;
+        private GVFSEnlistmentHealthCalculator.GVFSEnlistmentHealthData enlistmentHealthData;
+
+        [TestCase]
+        public void OnlyOneHydratedDirectory()
+        {
+            this.SetUpStructures();

```

Comment:
Nunit has the ability to specify a method that is run before / after each test. Can we use those here? This setup method looks more like a teardown / cleanup method:

https://github.com/nunit/docs/wiki/SetUp-and-TearDown

Reply:
OK cool I didn't know that! Got it working 👍